roles/netbox-static-data/tasks/main.yaml

```diff
@@ -67,3 +67,22 @@
       description: "{{item.description | default(item.prefix) }}"
     state: present
   loop: "{{ netbox_prefixes }}"
+
+- name: Create cluster type
+  netbox.netbox.netbox_cluster_type:
+    netbox_url: "{{ netbox_url }}"
+    netbox_token: "{{ netbox_token }}"
+    data:
+      name: "{{ item }}"
+    state: present
+  loop: "{{ netbox_cluster_types }}"
+
+- name: Create clusters
```

Comment:
I guess we would eventually register things like kaizen and the openshift environments as "clusters"?

Reply:
unless we want to track virtual machines running on those clusters, I don't see a reason to create those clusters here.

Reply:
Right, I got that from your response to my earlier question :)